var/spack/repos/builtin/packages/perl-module-build/package.py

```diff
@@ -36,6 +36,7 @@ class PerlModuleBuild(PerlPackage):
     """
 
     homepage = "http://search.cpan.org/perldoc/Module::Build"
-    url      = "http://search.cpan.org/CPAN/authors/id/L/LE/LEONT/Module-Build-0.4220.tar.gz"
+    url      = "http://search.cpan.org/CPAN/authors/id/L/LE/LEONT/Module-Build-0.4224.tar.gz"
 
+    version('0.4224', 'b74c2f6e84b60aad3a3defd30b6f0f4d')
```

Comment:
It's preferred to use sha256 for all new versions / packages. Could you please update ?